src/plugins/dev_tools/public/dev_tools_icon.test.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import { DevToolsIcon } from './dev_tools_icon';
+import { coreMock } from '../../../core/public/mocks';
+import { DEFAULT_NAV_GROUPS } from '../../../core/public';
+
+describe('<DevToolsIcon />', () => {
+  it('should call some methods from core service when click', () => {
```

Comment:
Nit: better to elaborate what's the `some methods`

Reply:
Sure, updated.